runtime/shiden/src/lib.rs

```diff
@@ -777,6 +799,145 @@ impl pallet_ethereum::Config for Runtime {
     type StateRoot = pallet_ethereum::IntermediateStateRoot<Self>;
 }
 
+parameter_types! {
+    pub CouncilMotionDuration: BlockNumber = 36 * HOURS;
```

Comment:
Should be 3 days according to our initial plan.

Reply:
Changed to 3 days